### DISCUSSION TITLE
Playing games via Sandboxie Plus?

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2510] [Q&A Feedback] [answered]
I have been wondering if it is possible to play games via Sandboxie Plus? I usually play singleplayer games which are 10-15-20 years old. I would like to launch them via Sandboxie Plus in order to maintain my OS intact. Some games install and leave a lot of stuff that is not necessary and simply clutter the drive. By launching them via Sandboxie I would be able to get rid of them with a single click, not worrying about some leftovers made by them. However, I'v been wondering if I should expect degradation of performance while playing a game via Sandboxie? Or maybe there are some other issues? Are you aware of settings for a sandbox that would make the performance hit as little as possible (if there's any)?

**Comment by Domochevsky** [comment]
Personally speaking, I play basically all my games within their own sandboxes for that specific reason (containment). They perform pretty well, all in all. If there's a performance hit then I have not noticed it yet. :)

That said, I do know that there's a mode within SB+ that basically says "containment only, no security". If in doubt you could turn that on.

**Reply by ghost** [reply]
There are two modes:
1. Application Compartment with Data Protection
2. Application Compartment (NO Isolation)

which one do you use? Which one would you suggest to use?
There's not much documentation available related to these modes, and which one of them would fit gaming in particular.

**Reply by Domochevsky** [reply]
Personally, I don't use either mode, as the default works well for me. I'm not familiar enough with either option to know what it would actually help with, internally. If the number of calls not being filtered somehow matter, and such. 

But I suspect either will be good for compatibility, if a game just doesn't want to play nice with default SB+.

**Comment by ghost** [comment]
@DavidXanatos any chance you could provide some insight into the best possible way of running/playing games via Sandboxie Plus without any possible performance loss (or any other issue)? Any details regarding the aforementioned compartment modes of a sandbox?

**Reply by ghost** [reply]
@DavidXanatos
I'd like to bump up the question I asked above, could you shed some light on the above-mentioned box presets? Which of them would be suitable for gaming?

**Reply by bastik-1001** [reply]
There is also https://github.com/sandboxie-plus/Sandboxie/issues/2855 which indicates that isolation has a greater impact for some games or that there is some other bottleneck that just kicks in when a certain condition is met.

**Comment by DavidXanatos** [comment, marked as answer]
Of caus you can run games sandboxed, for optimal performance the use of a compartment type box is advisable.
And as usual compatibility is never guaranteed, if it works it works if it does not it needs fixing, 
if it can be fixed depends on why it does not work, etc...

**Reply by ghost** [reply]
Could you elaborate more on differences between those two presets (with Data protection vs. No isolation)? There is no documentation available about it. Which one would be the most suitable for regular gaming purpose? All that I, and probably other users who would like to make use of Sandboxie for gaming, care about, is keeping game-related data (configs, saves, registry entries etc.) within one box while keeping OS intact. I don't consider games I'd like to play via Sandboxie as malware of suspicious in any other way, so I don't need virus-alike protection.
I realize there are games, especially the most recent competitive multiplayer games, which come bundled with numerous anticheat systems which install deep into OS. That's absolutely other use-case I'm not into.
What I am into is a plain simple box which contains a game and its related data and doesn't impact performance. If there hasn't been anything like this in Sandboxie yet, would it be possible for you to create it?

**Reply by ghost** [reply]
@DavidXanatos 
Any chance we ever get a possibility of disabling "Security Isolation" for a box in free version of Sandboxie Plus? I would like to test it out to find out if Sandboxie can really handle playing games without performance hit. According to a ticket - #2855 - it seems to be the culprit of serious performance hit.

**Reply by DavidXanatos** [reply]
@Mitch037 for testing you can request a 3 day test certificate by email.

**Comment by Mitch037** [comment]
An up-to-date list of games tested in Sandboxie Plus available at #4172.
